build.cake

```diff
@@ -141,6 +149,8 @@ Task("Test")
             NoBuild = true,
             NoRestore = true,
             Configuration = configuration,
+            Logger = "trx",
```

Comment:
Is this change actually related to CSHARP-1847?

Reply:
Removed.